### DISCUSSION TITLE
Can I get previous data?

### DISCUSSION THREAD

**Discussion by JongssLee** [ping-pub/explorer#271] [Q&A] [answered]
Can I get previous validator ranking data??
e.x) validator ranking for cosmos at 22-10-15

**Comment by liangping** [comment, marked as answer]
No.

**Reply by JongssLee** [reply]
Ok... Thx

**Reply by liangping** [reply]
https://{{rest_host}}/cosmos/staking/v1beta1/historical_info/{height} 

@JongssLee You can try this one,  but the height could be pruned if it's too old
